datasets/crd3/README.md

```diff
@@ -154,11 +174,11 @@ The data fields are the same among all splits.
 
 ### Dataset Curators
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+CRTranscript provided transcripts of the show; contributors of the Critical Role Wiki provided the abstractive summaries.
 
 ### Licensing Information
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+This work is licensed under a [Creative Commons Attribution-ShareAlike 4.0 International License][cc-by-sa]., as corresponding to the Critical Role Wiki https://criticalrole.fandom.com/
```

Comment:
In the tags it is written cc-by-4.0, while here it is written cc-by-sa. Which one is correct ?

Reply:
Thanks for catching that! `cc-by-sa-4.0` is the correct one. I've updated accordingly.